apps/sharedtest/test/unit/wifi_helper_test.js

```diff
@@ -5,6 +5,134 @@ require('/shared/js/wifi_helper.js');
 
 suite('WifiHelper', function() {
 
+  suite('> isValidInput()', function(){
+
+    test('Test open network >', function() {
+      var network_security = '';
+
+      assert.isTrue(WifiHelper.isValidInput(network_security, null, null),
+                        'Input should be valid if network is open');
+      assert.isTrue(WifiHelper.isValidInput(network_security, 'password', null),
+                        'Input should be valid if network is open');
```

Comment:
@sfoster I was just trying to make sure that WifiHelper.isValidInput always returns true with an open network. So can I remove these tests? Because actually these tests doesn't make sense, I just added it because  https://github.com/mozilla-b2g/gaia/blob/master/shared/js/wifi_helper.js#L143 that returns true if not using a network key.